README.md

```diff
@@ -44,6 +44,13 @@ You'll get an error in the end, but an executable will nevertheless be created i
 zip -j runtime.zip bootstrap /path/to/lumo runtime.cljs
 ```
 
+or
+
+```
+make clean (if necessary)
+env LUMO_BIN_PATH=/path/to/lumo make
```

Comment:
What does the `env` call do here? It seems to work for be by just specifying 

```
LUMO_BIN_PATH=/path/to/lumo make
```

Reply:
It is an alternative that doesn't the same 😁 I can get rid of it